webpack.config.js

```diff
@@ -94,7 +94,7 @@ module.exports = {
         analyzerMode: mode !== "development" ? "disabled" : "server",
       }),
     ].filter(Boolean),
-    devtool: dev && "inline-cheap-module-source-map",
+    devtool: dev ? "inline-cheap-module-source-map" : "source-map",
```

Comment:
Webpack doumentation suggests `source-map` as the recommended choice for production builds with high quality SourceMaps.